### ISSUE TITLE
Cannot run scdl: ImportError: cannot import name '__version__'

### ISSUE THREAD

**Issue by youurayy** [flyingrub/scdl#107]
Hi, I'm at loss here, tried to install both through pip and via clone, to no luck:

![image](https://cloud.githubusercontent.com/assets/588573/12752329/c0ba8506-c9ca-11e5-88eb-680e37cf9040.png)

After installing with pip, `scdl` is not even in the PATH, cannot run it. If I try to run it directly from python3.5/site-packages, it says the same as in the screenshot.

HALP !!!!

**Comment by youurayy** [comment]
`/usr/bin/env python3 --version` says `Python 3.5.1`

**Comment by youurayy** [comment]
OS X El Capitan 10.11.2, Python 3.5 installed via MacPorts.

**Comment by flyingrub** [comment]
Run `python3 setup.py install`

**Comment by youurayy** [comment]
Have run `python3 setup.py install`. Not in the PATH (easy to fix), but upon direct calling it's missing module 'six':

![image](https://cloud.githubusercontent.com/assets/588573/12754089/89bed00e-c9d2-11e5-947e-4cb88a7201f7.png)

**Comment by youurayy** [comment]
After installing `six` via pip, scdl appears to work. Thanks for the assistance.

As for the pip install - not sure where to look for the log.

**Comment by flyingrub** [comment]
I was just refering to the output of pip when you installed scdl :)
The six module should have been downloaded thanks to soundcloud-python (https://github.com/soundcloud/soundcloud-python/blob/master/requirements.txt#L5).
I don't know what happened.

**Comment by youurayy** [comment]
hey, same thing on windows 10, not sure what's wrong

![image](https://cloud.githubusercontent.com/assets/588573/14572076/9f0fd3fa-0301-11e6-91e4-ace45582c553.png)

**Comment by youurayy** [comment]
and again `pip3 install six`  fixes it:)

**Comment by grinapo** [comment]
Finished processing dependencies for scdl==1.6.9

```python3 setup.py install``` doesn't work, since it results the original
```ImportError: cannot import name __version__```

pip seem to install some more modules, but the result is the same. (python 3.6.3 pip 9.0.1)

**Comment by flyingrub** [comment]
Don't run the .py file directly. If you installed it correctly you just have to run `scdl [args]` from anywhere.

**Comment by grinapo** [comment]
I did't, and I did.

```
$ scdl
Traceback (most recent call last):
  File "/usr/local/bin/scdl", line 11, in <module>
    load_entry_point('scdl==1.6.9', 'console_scripts', 'scdl')()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 564, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2662, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2316, in load
    return self.resolve()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2322, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "build/bdist.linux-x86_64/egg/scdl/scdl.py", line 62, in <module>
ImportError: cannot import name __version__
```

**Comment by flyingrub** [comment]
You probably installed it using pip and not pip3. As you can see on the log you are trying to run it on python2...

**Comment by grinapo** [comment]
May be true and false. I started ```python3 setup.py``` but it's true that python2 is the default on the system. Then I have used pip3. I am not familiar enough with python environment to see what does it screw and how, but did it with the best intentions. ;-) 

So, based on your valuable feedback I observed that the setup.py have created /usr/local/bin/scdl while pip3 crated ~/.local/bin/scdl, and obviously only one can be picked to run. Seems pip3 was on the losing side. Removing the /usr/local one results no scdl in the path but it's working when I call it directly from ~/.local/.  Manual symlinking ftw. _Thanks._
